apps/bam.iobio/src/App.tsx

```diff
@@ -69,6 +70,9 @@ const bamFileStats = (bamFile: string | null) => {
 };
 
 function App() {
+	// TODO: IobioComponents is needed in scope for web components but linter removes unused imports
+	console.log(IobioComponents);
```

Comment:
most linter have an ignore pragma comment thingy you can probably use
eg. above the import it would likley be something like `// @es-lint-ignore`

Reply:
This is already fixed by `import '@overture-stack/iobio-components/components/';`